src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/Contact/ContactType.php

```diff
@@ -95,27 +99,31 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
             ->add('email', EmailType::class, [
+                'label' => $this->trans('Email address', 'Admin.Global'),
+                'help' => $this->trans('Emails will be sent to this address.', 'Admin.Shopparameters.Help'),
                 'required' => false,
                 'constraints' => [
                     new Email([
                         'message' => $this->trans(
                             '%s is invalid.',
-                            [],
                             'Admin.Notifications.Error'
                         ),
                     ]),
                 ],
             ])
-            ->add('is_messages_saving_enabled', SwitchType::class)
+            ->add('is_messages_saving_enabled', SwitchType::class, [
+                'label' => $this->trans('Save messages?', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans('If enabled, all messages will be saved in the "Customer Service" page under the "Customer" menu.', 'Admin.Shopparameters.Help'),
+            ])
             ->add('description', TranslatableType::class, [
+                'label' => $this->trans('Description', 'Admin.Global'),
                 'type' => TextareaType::class,
                 'required' => false,
                 'options' => [
                     'constraints' => [
-                        new CleanHtml([
+                        new NoTags([
                             'message' => $this->trans(
-                                '%s is invalid.',
-                                [],
+                                'The field %s is invalid. HTML tags are not allowed.',
```

Comment:
```suggestion
                                'The "%s" field is invalid. HTML tags are not allowed.',
```
So I think we should keep it to stay consistent with other PRs.